<?xml version="1.0"?>
<launch>
    <!-- 加载目标车辆的键盘控制节点 -->
    <node pkg="simulation" type="keyboard_control" name="keyboard_control" output="screen">
        <remap from="cmd_vel" to="/target_car/cmd_vel"/>
    </node>
    
    <!-- 加载参数 -->
    <rosparam command="load" file="$(find track)/config/tracking_params.yaml" />
    
    <!-- 启动跟踪节点 -->
    <node name="tracking_node" pkg="track" type="tracking_node" output="screen">
        <param name="max_linear_speed" value="0.8"/>
        <param name="max_angular_speed" value="1.5"/>
        <param name="min_distance" value="0.5"/>
        <param name="max_distance" value="2.0"/>
        <param name="angle_threshold" value="0.1"/>
    </node>
</launch>
